datasets/chorus-dt/utils/resolvers.py

```diff
@@ -68,33 +68,45 @@ def __init__(self):
         self.insee_codes = self.load_codes("./data/prepared/insee_codes.pkl")
         self.tvs_codes = self.load_codes("./data/prepared/tvs_codes.pkl")
         self.uic_codes = self.load_codes("./data/prepared/uic_codes.pkl")
+        self.iata_codes = self.load_codes("./data/prepared/iata_codes.pkl")
 
     def load_codes(self, path):
         with open(path, "rb") as file_id:
             return dill.load(file_id)
 
     def resolve(self, x):
         code_2_as_int = self.get_int(x["code_2"])
-        if code_2_as_int and (code_2_as_int in self.uic_codes["lon"]):
-            x["lon"] = self.uic_codes["lon"][code_2_as_int]
-            x["lat"] = self.uic_codes["lat"][code_2_as_int]
-            x["resolved"] = True
-            x["resolved_through_uic_code"] = True
-        elif x["code_1"] in self.insee_codes["lon"]:
-            x["lon"] = self.insee_codes["lon"][x["code_1"]]
-            x["lat"] = self.insee_codes["lat"][x["code_1"]]
-            x["resolved"] = True
-            x["resolved_through_insee_code"] = True
-        elif x["place"] == '-75056': # Hard exception...
-            x["name"] = 'Paris'
-        # elif x['code_1'] in TVS_CODES['lon']:
-        #     x['lon'] = TVS_CODES['lon'][x['code_1']]
-        #     x['lat'] = TVS_CODES['lat'][x['code_1']]
-        #     x['resolved'] = True
-        #     x['resolved_through_tvs_code'] = True
-        else:
-            x["lon"] = None
-            x["lat"] = None
+        
+        try:
+            if code_2_as_int and (code_2_as_int in self.uic_codes["lon"]):
+                x["lon"] = self.uic_codes["lon"][code_2_as_int]
+                x["lat"] = self.uic_codes["lat"][code_2_as_int]
+                x["resolved"] = True
+                x["resolved_through_uic_code"] = True
+            elif x["code_1"] in self.insee_codes["lon"]:
+                x["lon"] = self.insee_codes["lon"][x["code_1"]]
+                x["lat"] = self.insee_codes["lat"][x["code_1"]]
+                x["resolved"] = True
+                x["resolved_through_insee_code"] = True
+            elif ((str(x["code_1"][-3:]) in self.iata_codes["lon"]) and
+                 (str(x["code_1"][:2]) == 'IA')):                
+                x["lon"] = self.iata_codes["lon"][x["code_1"][-3:]]
+                x["lat"] = self.iata_codes["lat"][x["code_1"][-3:]]
+                x["resolved"] = True
+                x["resolved_through_iata_code"] = True
+            elif x["place"] == '-75056': # Hard exception...
+                x["name"] = 'Paris'
+            # elif x['code_1'] in TVS_CODES['lon']:
+            #     x['lon'] = TVS_CODES['lon'][x['code_1']]
+            #     x['lat'] = TVS_CODES['lat'][x['code_1']]
+            #     x['resolved'] = True
+            #     x['resolved_through_tvs_code'] = True
+            else:
+                x["lon"] = None
+                x["lat"] = None
+                
+        except:
```

Comment:
As it will be used for debug, just catch an `Exception` and print it, otherwise it might be hard to understand the errors,
```suggestion
        except Exception as e:
            print(e)
```